src/ngx_http_lua_regex.h

```diff
@@ -14,6 +14,7 @@
 
 #if (NGX_PCRE)
 void ngx_http_lua_inject_regex_api(lua_State *L);
+ngx_int_t ngx_http_lua_ffi_set_jit_stack_size(int size);
```

Comment:
Because this C API is part of the ABI, I think we should always make this C API function present but always return an error in that C function when PCRE JIT is not present.